opentracing-spring-cloud/pom.xml

```diff
@@ -22,6 +22,11 @@
       <artifactId>opentracing-spring-web-autoconfigure</artifactId>
       <version>${version.io.opentracing.contrib-opentracing-spring-web}</version>
     </dependency>
+    <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-jdbc</artifactId>
```

Comment:
It should exclude all transitive dependencies

Reply:
```
[INFO] +- io.opentracing.contrib:opentracing-jdbc:jar:0.0.3:compile                                                                                                                                                
[INFO] |  \- io.opentracing:opentracing-util:jar:0.30.0:compile                                                                                                                                                    
```

The only one is the opentracing-util. Should I really exclude it?

Reply:
I think that is fine.